### PR TITLE
v1.17 changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,23 @@
 # Mattermost Mobile Apps Changelog
 
-## 1.16.1 Release
-- Release Date: February 21, 2019
+## 1.17.0 Release
```
Comment (enahum) [inline CHANGELOG.md]: We should highlight that we upgraded to RN 0.58.5 and that the iOS Share Extension is now built natively

Comment (enahum) [reply CHANGELOG.md]: Oh and btw this should be on top of the 1.16.1 release

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
 - Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
 
 ### Combatibility
  - Mobile App v1.13+ is required for Mattermost Server v5.4+.
  - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
 
 ### Bug Fixes
- - Fixed an issue where link previews and reactions weren't displayed when post metadata was disabled.
- - Fixed an issue on Android where the app crashed when sharing multiple files.
+ - Fixed an issue where user profile icon for a Webhook did not follow use_user_icon.
```
Comment (enahum) [inline CHANGELOG.md]: really important to list the crash fixes

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,31 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.17.0 Release
+- Release Date: March 16, 2019
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+ - Mobile App v1.13+ is required for Mattermost Server v5.4+.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
```
Comment (saturninoabril) [inline CHANGELOG.md]: It seems we're not doing in the past but should we do the same compatibility notice for iOS device and min. OS version?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,31 @@
...
+ - Fixed an issue where user profile icon for a Webhook did not follow use_user_icon.
+ - Fixed an issue where emoji not included in our list of text-based emoji were not rendered as jumboemoji.
+ - Fixed an issue where notification badges were not cleared from other clients when channel was changed via push notification after the mobile app was opened.
+ - Fixed an issue where the app did not show local notification when session expired.
+ - Fixed an issue where the "No Internet Connection" banner did not always display when internet connectivity was lost.
+ - Fixed an issue where opening channels with unreads had loading indicator placed above unread messages line.
+ - Fixed an issue where jumbo-emoji-only reply sometimes appeared with a large space beneath.
+ - Fixed an issue where the "No Internet Connection" banner did not always disappear when connection was re-estabilished.
+ - Fixed an issue where the profile picture for webhooks was showing the hook owner picture.
+ - Fixed an issue where email notifications setting was out of sync with the webapp until the setting was edited.
```
Comment (saturninoabril) [inline CHANGELOG.md]: Can we rearrange the list based on similarities, like for profile icon/picture for webhook, jumbo emoji, notification and internet connection?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,33 @@
...
+- Release Date: March 16, 2019
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+ - If **DisableLegacyMfa** setting in ``config.json`` is set to ``true`` and [multi-factor authentication](https://docs.mattermost.com/deployment/auth.html) is enabled, ensure your users have upgraded to mobile app version 1.17 or later. See [Important Upgrade Notes](https://docs.mattermost.com/administration/important-upgrade-notes.html) for more details.
+ - Mobile App v1.13+ is required for Mattermost Server v5.4+.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+ 
+### Highlights
```
Comment (esethna) [inline CHANGELOG.md]: Not necessary to highlight these as there is no change to the user

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,33 @@
...
+ - Upgraded mobile apps to RN 0.58.5.
+ - iOS Share Extension is now built natively.
+
+### Bug Fixes
+ - Fixed several Android app crash / fatal error issues.
+ - Fixed an issue on Android where the app crashed intermittently when selecting a link.
+ - Fixed an issue where email notifications setting was out of sync with the webapp until the setting was edited.
+ - Fixed an issue where notification badges were not cleared from other clients when channel was changed via push notification after the mobile app was opened.
+ - Fixed an issue where the app did not show local notification when session expired.
+ - Fixed an issue where user profile icon for a webhook did not follow use_user_icon.
```
Comment (esethna) [inline CHANGELOG.md]: Not sure what is the difference between this and the one below? Can we make this more readable?

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,33 @@
...
+ - Fixed an issue where the profile picture for webhooks was showing the hook owner picture.
+ - Fixed an issue where emoji not included in our list of text-based emoji were not rendered as jumboemoji.
```
Comment (esethna) [inline CHANGELOG.md]: we should aim for these to be readable by someone who isn't a dev.

ie 
Fixed an issue where some emoji were not rendered as jumbo.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,33 @@
...
+ - Fixed an issue where jumbo-emoji-only reply sometimes appeared with a large space beneath.
```
Comment (esethna) [inline CHANGELOG.md]: same here, needs to be more readable:

Fixed an issue where jumbo emoji posted as a reply sometimes appeared with large space beneath.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,33 @@
...
+ 
+### Highlights
+ - Upgraded mobile apps to RN 0.58.5.
+ - iOS Share Extension is now built natively.
+
+### Bug Fixes
+ - Fixed several Android app crash / fatal error issues.
+ - Fixed an issue on Android where the app crashed intermittently when selecting a link.
+ - Fixed an issue where email notifications setting was out of sync with the webapp until the setting was edited.
+ - Fixed an issue where notification badges were not cleared from other clients when channel was changed via push notification after the mobile app was opened.
```
Comment (esethna) [inline CHANGELOG.md]: Not sure what "channel was changed via push notification" means

File: CHANGELOG.md (hunk 5 above)
Comment (esethna) [reply CHANGELOG.md]: 1.17 was mainly just bug fixes. No highlights required

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,33 @@
...
+ - Mobile App v1.13+ is required for Mattermost Server v5.4+.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+ 
+### Highlights
+ - Upgraded mobile apps to RN 0.58.5.
+ - iOS Share Extension is now built natively.
+
+### Bug Fixes
+ - Fixed several Android app crash / fatal error issues.
```
Comment (esethna) [inline CHANGELOG.md]: First let's add this one:

- Fixed support for EMM connections using VPN on-demand. See docs for more details on [setting AppConfig values](https://docs.mattermost.com/mobile/mobile-appconfig.html#mattermost-appconfig-values) for VPN support.

File: CHANGELOG.md (hunk 5 above)
Comment (amyblais) [reply CHANGELOG.md]: Elias said that these should be highlighted. @enahum What do you think?

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,33 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.17.0 Release
+- Release Date: March 16, 2019
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+ - If **DisableLegacyMfa** setting in ``config.json`` is set to ``true`` and [multi-factor authentication](https://docs.mattermost.com/deployment/auth.html) is enabled, ensure your users have upgraded to mobile app version 1.17 or later. See [Important Upgrade Notes](https://docs.mattermost.com/administration/important-upgrade-notes.html) for more details.
```
Comment (esethna) [inline CHANGELOG.md]: Is the important upgrade notes doc updated? Don't see anything recent in there for this

Comment (amyblais) [reply CHANGELOG.md]: Yes... v5.9 docs haven't been merged yet as we're not ready to cut final yet

Comment (esethna) [reply CHANGELOG.md]: And to verify, is this setting enabled or disabled by default in v5.9? I think disabled?

Comment (amyblais) [reply CHANGELOG.md]: Disabled, but it's still good to have the note if admins change it to True

Comment (jasonblais) [inline CHANGELOG.md]: There is one more compatibility note for customers as a result of #2627 

https://community.mattermost.com/core/pl/u8g8dt4z1jbhij1c6xj7krnfdy

/cc @esethna

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,34 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.17.0 Release
+- Release Date: March 16, 2019
```
Comment (enahum) [inline CHANGELOG.md]: date should be updated

File: CHANGELOG.md (hunk 11 above)
Comment (enahum) [reply CHANGELOG.md]: @jasonblais I've believe the one you are talking about is right below this one

File: CHANGELOG.md (hunk 5 above)
Comment (enahum) [reply CHANGELOG.md]: The Share Extension was re-done from the ground up, so I do think it should be highlighted. the upgrade to RN 0.58.5 is of no importance

File: CHANGELOG.md (hunk 11 above)
Comment (amyblais) [reply CHANGELOG.md]: I just added it

**Comment by amyblais** [comment]
I've added all bug fixes except bugs that were found on RCs

**Comment by amyblais** [comment]
@enahum / @esethna Can you help review / merge this by today? (I don't have merge permissions)

File: CHANGELOG.md (hunk 5 above)
Comment (esethna) [reply CHANGELOG.md]: ya it was rebuilt but there's no change to the end user right? So why highlight it?

**Pull request merged**